tests/blob-api.js

```diff
@@ -0,0 +1,122 @@
+import { join, basename } from 'node:path'
+import { fileURLToPath } from 'url'
+import test from 'brittle'
+import { BlobApi } from '../src/blob-api.js'
+import { createBlobServer, getPort } from '../src/blob-server/index.js'
+import { createBlobStore } from './helpers/blob-store.js'
+import { timeoutException } from './helpers/index.js'
+
+test('get port after listening event with explicit port', async (t) => {
```

Comment:
I think this is tested in the getUrl test below, which avoids needing to test the `getPort`, which is an implementation detail.